app/content/models/external_controller01_left.mtl

```diff
@@ -0,0 +1,13 @@
+# Blender MTL File: 'None'
+# Material Count: 1
+
+newmtl blinn1SG
+Ns 96.078431
+Ka 0.000000 0.000000 0.000000
+Kd 0.900000 0.900000 0.900000
+Ks 0.400000 0.400000 0.400000
+Ke 0.000000 0.000000 0.000000
+Ni 1.000000
+d 1.000000
+illum 2
+map_Kd C:\Users\Nicolas\Desktop\external_controllers\external_controller01_col.png
```

Comment:
blender's obj exporter...
No need to worry about that, the texture is being set in code